### PR TITLE
Add scaffolding for contact form

### PR DESCRIPTION
**Pull request by Edward-Kabue**
**Collaborate with team to :**
- Add Contact form.
- Style the form
- Add formspree action

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -252,6 +252,64 @@ <h2 class="connect">
...
+        <div class="contact-form">
+
+            <form action="https://formspree.io/f/mjvlwyga" method="post">
+                <ul>
+                    <li>
+                        <input type="text" id="name" name="user_name" maxlength="30" required>
+                    </li>
+                    <li>
+                        <input type="email" id="mail" name="user_email" required>
+                    </li>
```
Comment (gemmen29) [inline index.html]: Kindly add a placeholder for the name and the email.

File: index.html [hunk 2]
```diff
@@ -252,6 +252,64 @@ <h2 class="connect">
...
+                    <li>
+                        <textarea id="msg" name="user_message" maxlength="500"
+                            placeholder="Write your message here"></textarea>
```
Comment (gemmen29) [inline index.html]: Please add the `required` attribute for the ` textarea `.

File: index.html [hunk 3]
```diff
@@ -252,6 +252,64 @@ <h2 class="connect">
             </section>
         </section>
     </section>
+    <section class="contact">
+        <div class="texted">
+            <img src="./images/Ellipse-contact.svg" alt="ec">
+
```
Comment (Regiss05) [inline index.html:258]: Please can you remove this empty space? 👍

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi @Edward-Kabue, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 4]
```diff
@@ -252,6 +252,64 @@ <h2 class="connect">
...
+        </div>
+        <div class="last">
+            <img src="./images/Ellipse-4.svg" alt="eclipseo">
+        </div>
+        <div class="lastone">
+            <img src="./images/Ellipse-13.svg" alt="exl">
+        </div>
+        <div class="lp">
+            <img src="./images/Ellipse-8.svg" alt="inb">
+        </div>
```
Comment (Regiss05) [inline index.html:280]: - [ ] Kindly replace the div tag with a list tag 👌🏽

**Review by gemmen29 — APPROVED** [review]
Hi @Edward-Kabue, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Regiss05 — CHANGES_REQUESTED** [review]
Hi @Edward-Kabue

**Good job so far!**
There are some issues that you still need to work on to go to the next project but you are almost there!
clapping

congratulations-gif

### **To highlight 💯**
Good commit message ✔️
No Linter errors ✔️
Follow Github flow ✔️

### **Required Changes ♻️**
_Check the comments under the review._

### **Optional suggestions**
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

_Cheers and Happy coding!👏👏👏_

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on @Regiss05 in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

File: index.html (hunk 4 above)
Comment (Edward-Kabue) [reply index.html:280]: The above images are the background  images

**Review by Fsher07 — COMMENTED** [review]
You did a great job, congratulations 🎉
giphy

**Pull request merged**